companion/src/translations/companion_de.ts

```diff
@@ -1134,32 +1135,32 @@ Do you want to import them?</source>
     <message>
         <location filename="../creditsdialog.cpp" line="95"/>
         <source>Main developers</source>
-        <translation type="unfinished"></translation>
+        <translation>Hauptentwickler</translation>
     </message>
     <message>
         <location filename="../creditsdialog.cpp" line="97"/>
         <source>Translators</source>
-        <translation type="unfinished"></translation>
+        <translation>Übersetzer</translation>
     </message>
     <message>
         <location filename="../creditsdialog.cpp" line="99"/>
         <source>Companies and projects who have donated to OpenTX</source>
-        <translation type="unfinished"></translation>
+        <translation>Firmen und Projekte die für OpenTX gespendet haben</translation>
```

Comment:
ich würde das für weglassen

Reply:
Hm. Ich denke, dass das "für" schon richtig ist. Im Englischen heißt es ja auch "to OpenTX". Auf jeden Fall muss aber noch ein Komma nach Projekte hin ;)